platform/admin/ui/src/main/webapp/templates/module/ModuleDetail.layout.handlebars

```diff
@@ -14,7 +14,7 @@
 <div class="page-layout full-height well">
     <div class="header clearfix">
         <div class="btn btn-success pull-right">
-            Export
+            Export System Configurations
```

Comment:
Since we're under the `System` tab, do we need to repeat `System` in the button label? Also, should we remove the `s` in `Configurations`?